guides/common/modules/proc_salt-installing-salt-plugin.adoc

```diff
@@ -0,0 +1,28 @@
+[id="salt_guide_installing_salt_plugin_{context}"]
+= Installing Salt Plugin
+
+In order to configure managed hosts with Salt, you need to install the Salt plugin.
+
+ifdef::orcharhino[]
+[TIP]
+====
+Select *Salt* as a configuration management system during step five of the xref:sources/installation_and_maintenance/orcharhino_installation_guide.adoc#oig_main_installation_steps[main orcharhino installation steps].
+Choosing this option installs and configures both the Salt plugin and a Salt Master on your orcharhino.
+====
+endif::[]
```

Comment:
Warning: TIP

Reply:
IMO: if he wraps it for orcharhino, he's free to do what he likes without comment from me :dancers:

Reply:
No change.